docs/src/further_topics/index.rst

```diff
@@ -13,7 +13,7 @@ that may be of interest to the more advanced or curious user.
 .. hint::
 
    If you wish further documentation on any specific topics or areas of Iris
-   that are missing, then please let us know by raising a `GitHub Documentation Issue`_
+   that are missing, then please let us know by raising a :issue:`GitHub Documentation Issue<new/choose>`
```

Comment:
@rcomer That's the `rst` pattern I didn't manage to find and dodged... nice one :partying_face: